packages/components/src/Switch/Switch.tsx

```diff
@@ -39,7 +39,7 @@ export function Switch({
     }
   };
 
-  const className = classnames(styles.track, {
+  const className = classnames(styles.track, styles.switch, {
```

Comment:
technically I could have done the styling on `track` but I feel like defining the variables on a class named the same as the component name is a more intuitive pattern.